src/core/views.py

```diff
@@ -2712,7 +2713,7 @@ def create_proposal_form(proposal):
     data = json.loads(proposal.data)
 
     for relation in relations:
-        v = data.get(relation.element.name)
+        v = data.get(relation.element.name.encode('ascii', 'xmlcharrefreplace'))
```

Comment:
You're doing this over and over throughout this process. Wouldn't it be useful to create a single method that does this, limiting the possibility of getting `.encode('ascii', 'xmlcharrefreplace')` wrong with any of the points it's used?